src/main/java/org/interledger/ilp/exceptions/InterledgerException.java

```diff
@@ -1,36 +1,137 @@
 package org.interledger.ilp.exceptions;
 
+import java.util.ArrayList;
+import java.time.ZonedDateTime;
+
+import org.interledger.ilp.InterledgerAddress;
+
 /**
  * Base ILP exception.
  *
- * @author Manuel Polo [mistermx@gmail.com]
+ * RFC REF: https://github.com/interledger/rfcs/blob/master/0003-interledger-protocol/0003-interledger-protocol.md
```

Comment:
We should point to the public URL: https://interledger.org/rfcs/0003-interledger-protocol/#errors